uberon-base.owl

```diff
@@ -43824,26 +43832,15 @@
                 <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/UBERON_0000066"/>
             </owl:Restriction>
         </rdfs:subClassOf>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0001009"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0001016"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0001033"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0001434"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0002204"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0002294"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0002330"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0002390"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0002405"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0002416"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0002423"/>
-        <owl:disjointWith rdf:resource="http://purl.obolibrary.org/obo/UBERON_0004456"/>
```

Comment:
lots of disjointnesses remove, intended?

Reply:
@bvarner-ebi & @nicolevasilevsky - I assume this is the work you guys did with the renal system stuff? could you check this please

Reply:
@shawntanzk, sorry, these do not look familiar.

Reply:
actually it might be @balhoff PR (https://github.com/obophenotype/uberon/pull/2530) - this looks like part of a release process too, which begs the question, why is that changed now? perhaps this got added while I was running the release. I'll try to rerun it to make sure it's up to date with that PR merge

Reply:
wait no, the order is right - shouldn't be syncing error

Reply:
sorry @balhoff - ignore that, was looking at the wrong thing >.<